docs/concepts/configuration/assign-pod-node.md

```diff
@@ -190,13 +190,7 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
-In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limited set of topology keys are allowed.
-By default the allowed topology keys are:
-
-* `kubernetes.io/hostname`
-* `failure-domain.beta.kubernetes.io/zone`
-* `failure-domain.beta.kubernetes.io/region`
+The `topologyKey` can be any legal label value.
```

Comment:
@kevin-wangzefeng I saw that we have the admission control here https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/antiaffinity/admission.go#L70 , the question is if I do not use `antiaffinity` control, I think that the `affinity` will work even without the `kubernetes.io/hostname` topology key?

@fleeto Did you enable `antiaffinity` admission control in api server?